qt/scientific_interfaces/Indirect/IndirectSqwModel.cpp

```diff
@@ -150,9 +150,68 @@ UserInputValidator IndirectSqwModel::validate(std::tuple<double, double> const q
   return uiv;
 }
 
-std::pair<double, double> IndirectSqwModel::roundToWidth(std::tuple<double, double> const &axisRange, double width) {
-  return std::make_pair(roundToPrecision(std::get<0>(axisRange), width) + width,
-                        roundToPrecision(std::get<1>(axisRange), width) - width);
+std::string IndirectSqwModel::getEFixedFromInstrument(std::string const &instrumentName, std::string analyser,
+                                                      std::string const &reflection) {
+
+  // In the IRIS IPF there is no fmica component
+  if (instrumentName == "IRIS" && analyser == "fmica")
+    analyser = "mica";
```

Comment:
should we give a warning to the users that it has been changed?

Reply:
We have not given a warning before this point.